packages/edge-bundler/package.json

```
@@ -37,24 +27,17 @@
     "test:integration": "node --experimental-modules test/integration/test.js",
     "vendor": "deno vendor --force --output deno/vendor https://deno.land/x/eszip@v0.55.2/mod.ts https://deno.land/x/retry@v2.0.0/mod.ts https://deno.land/x/std@0.177.0/path/mod.ts"
   },
-  "config": {
-    "eslint": "--ignore-path .gitignore --cache --format=codeframe --max-warnings=0 \"{node,scripts,.github}/**/*.{js,ts,md,html}\" \"*.{js,ts,md,html}\"",
-    "prettier": "--ignore-path .gitignore --loglevel=warn \"{node,scripts,.github}/**/*.{js,ts,md,yml,json,html}\" \"*.{js,ts,yml,json,html}\" \".*.{js,ts,yml,json,html}\" \"!**/package-lock.json\" \"!package-lock.json\" \"!node/vendor/**\""
-  },
   "keywords": [],
   "license": "MIT",
-  "repository": "https://github.com/netlify/edge-bundler",
+  "repository": "https://github.com/netlify/build",
```

For monorepos it's recommended to specify the path inside the repository:

```suggestion
  "repository": {
    "type": "git",
    "url": "https://github.com/netlify/build.git",
    "directory": "packages/edge-bundler"
  },  
```

---

addressed in https://github.com/netlify/build/pull/5599/commits/c5559a8250fda650ebb8f26744bac039378533d0